db-service/lib/cqn2sql.js

```diff
@@ -592,24 +592,64 @@ class CQN2SQLRenderer {
    * @returns {string} SQL
    */
   STREAM(q) {
```

Comment:
If we support `Readable` as values in `UPDATE`, `INSERT` and `UPSERT`. It should remove the need for `STREAM.into` cases. Which is why I changed the current internal logic of the STREAM handler to just translate them to the other types.

The only use case case I could for the STREAM event is reading a whole result set as raw json stream. This could also be done by adding a flag on the SELECT statement.

Additionally it would be required to adjust the `all` API so that the driver has a way of knowing what columns are of type LargeBinary as they should be returned as streams rather then full 2gib buffers.